src/test/java/com/epam/tc/hw3/components/CheckBoxRow.java

```diff
@@ -0,0 +1,46 @@
+package com.epam.tc.hw3.components;
```

Comment:
you should move all your page objects from tests package to src/main/java/(your_packages_for_PO)